packages/core/src/electron-browser/window/electron-window-service.ts

```diff
@@ -33,14 +35,32 @@ export class ElectronWindowService extends DefaultWindowService {
      */
     protected closeOnUnload: boolean = false;
 
+    /**
+     * Most recent value set for window.zoomLevel preference
+     */
+    protected prevPreferredZoomLevel: number | undefined;
```

Comment:
I do not believe it is necessary anymore, if there is a preference change event we should simply update the `zoom`.